models/public/hbonet-0.25/hbonet-0.25.md

```diff
@@ -0,0 +1,63 @@
+# hbonet-0.25
+
+## Use Case and High-Level Description
+
+The `hbonet-0.25` model is one of the classification models from https://github.com/d-li14/HBONet with `width_mult=0.25`
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 0.037         |
+| MParams           |               |
+| Source framework  | PyTorch\*     |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 57.30          |                 |
+| Top 5  | 79.80          |                 |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `input`, shape: [1x3x224x224], format: [BxCxHxW], where:
+
+- B - batch size
+- H - image height
+- W - image width
+- C - number of channels
```

Comment:
Please, align according to format BxCxHxW

Reply:
done, thanks!